frontend/amundsen_application/static/js/pages/ProfilePage/index.tsx

```diff
@@ -157,8 +162,6 @@ export class ProfilePage extends React.Component<
     );
   };
 
-  generateTabKey = (resource: ResourceType) => `tab:${resource}`;
```

Comment:
Are we sure this `tab:` part wasn't used by anything?

Reply:
There was a test for it but I deleted this. `tab:${resource}` shows up in the URL as `?tab=tab%3Atable` which isn't pretty.

Reply:
Cool, just making sure it wasn't important